README.rst

```diff
@@ -40,27 +40,45 @@ A minimal Quart example is,
 
 .. code-block:: python
 
-    from quart import Quart, websocket
+   from quart import Quart, websocket
+   from time import sleep
+   app = Quart(__name__)
 
-    app = Quart(__name__)
+   @app.route('/')
+   async def hello():
+      return 'hello'
 
-    @app.route('/')
-    async def hello():
-        return 'hello'
+   @app.websocket('/ws')
+   async def ws():
+      for i in range(5):
+         await websocket.send(f'hello {i}')
+         sleep(2)
 
-    @app.websocket('/ws')
-    async def ws():
-        while True:
-            await websocket.send('hello')
-
-    app.run()
+   app.run()
 
 if the above is in a file called ``app.py`` it can be run as,
 
 .. code-block:: console
 
     $ python app.py
 
+Then run the following html file to test
```

Comment:
I'm not sure this is required in the Quickstart (also as the code doesn't use it), the tutorials should cover this.

Reply:
the code doesn't use it ?

This previous example result is done by flask itself.